### ISSUE TITLE
User is not created 

### ISSUE THREAD

**Issue by dannoe** [microsoft/navcontainerhelper#1749]
**Describe the issue**
The new database created by the script below doesn't contain any user.

**Scripts used to create container and cause the issue**
```
$containerName = 'CONT1'
$credential = Get-Credential -Message 'Using UserPassword authentication. Please enter credentials for the container.'
$auth = 'UserPassword'
$artifactUrl = Get-BcArtifactUrl -type 'OnPrem' -version '17.4' -country 'de' -select 'Latest'
$databaseServer = 'host.containerhelper.internal'
$databaseInstance = ''
$databasePrefix = 'PREFIX'
$databaseName = 'BC'
$databaseUsername = ''
$databasePassword = ''
$databaseSecurePassword = ConvertTo-SecureString -String $databasePassword -AsPlainText -Force
$databaseCredential = New-Object pscredential $databaseUsername, $databaseSecurePassword
$licenseFile = 'C:\temp\license.flf'
New-BcContainer `
    -accept_eula `
    -containerName $containerName `
    -credential $credential `
    -auth $auth `
    -artifactUrl $artifactUrl `
    -imageName '170cu4-image' `
    -databaseServer $databaseServer -databaseInstance $databaseInstance -databaseName $databaseName -databasePrefix $databasePrefix `
    -databaseCredential $databaseCredential `
    -licenseFile $licenseFile `
    -updateHosts `
    -replaceExternalDatabases
```

**Full output of scripts**
```
Not necessary
```

**Screenshots**

**Additional context**
- does it happen all the time? 
Yes
- did it use to work?
Dunno.

**Comment by dannoe** [comment]
It's not running the following if block, because $artifactUrl is indeed empty.
https://github.com/microsoft/navcontainerhelper/blob/419f584c41bc7526be825d7b48f4e792ea2440c5/ContainerHandling/New-NavContainer.ps1#L2008

The variable is empty because it will be set to empty if ArtifactUrl and ImageName is specified:
https://github.com/microsoft/navcontainerhelper/blob/419f584c41bc7526be825d7b48f4e792ea2440c5/ContainerHandling/New-NavContainer.ps1#L559

**Comment by freddydk** [comment]
Based on the two bugs, it is clear that people normally don't create the image when using -replaceexternaldatabases, without the imagename, it would work.

It still should work with the imagename of course, will submit a fix in a bit

**Comment by dannoe** [comment]
Thanks again for the quick fix.

**Comment by freddydk** [comment]
Shipped in BcContainerHelper 2.0.6
